pkg/edit/editor_test.go

```diff
@@ -37,6 +37,37 @@ func TestEditor_DoesCommandWithLeadingSpaceToHistory(t *testing.T) {
 	testCommands(t, f.Store /* no commands */)
 }
 
+func TestEditor_FiltersCommandBeforeAddingToHistory(t *testing.T) {
+	noHistory := "_EMPTY_"
+
+	test := func(c, input, exp string) {
```

Comment:
If you let this function take variadic arguments and pass that to `testCommands`, you don't need to special-case `noHistory`.